### DISCUSSION TITLE
Append new line items to the cart instead of prepending

### DISCUSSION THREAD

**Discussion by benface** [craftcms/commerce#2452] [Ideas]
**What are you trying to do?**
A client is requesting that products should be added to the *end* of the cart (after the existing line items), not at the beginning, which is the current behavior (and it seems [hardcoded that way](https://github.com/craftcms/commerce/blob/main/src/elements/Order.php#L1662)).

**What's your proposed solution?**
A new Commerce config setting, like `appendLineItems` (`false` by default).

**Comment by lukeholder** [comment]
@benface I will leave this as an open feature request, and see if anyone else really needs this.

Amazon prepends to the item to the cart when you click 'Add to cart'.

In the mean-time, you can just do `cart.lineItems|reverse` when you loop over the line items within your templates to get them in the order you expect.
